ajax/libs/jQuery-Paging/1.2.0/jquery.paging.js

```diff
@@ -0,0 +1,545 @@
+/**
+ * @license jQuery paging plugin v1.3.0 23/06/2014
```

Comment:
Shouldn't here be v1.2.0? I thought you added a wrong file.
https://github.com/infusion/jQuery-Paging/blob/v1.2.0/jquery.paging.js#L2